src/enums.rs

```diff
@@ -103,6 +125,29 @@ pub fn int_to_ev_rel(code: u32) -> Option<EV_REL> {
     }
 }
 
+impl std::str::FromStr for EV_REL {
+    type Err = ();
```

Comment:
I haven't written enough Rust to know what best practice is here. But there isn't really any useful error info we can hand back to the caller.

Reply:
I agree there is not much that we can return to the user.